sentry/src/main/java/io/sentry/TransactionOptions.java

```diff
@@ -196,4 +200,14 @@ public void setAppStartTransaction(final boolean appStartTransaction) {
   public boolean isAppStartTransaction() {
     return isAppStartTransaction;
   }
+
+  @ApiStatus.Internal
+  public @Nullable ISpanFactory getSpanFactory() {
```

Comment:
This is required to override the span factory when creating Sentry transactions in OTel `SpanExporter` to avoid endless loops.